index.html

```diff
@@ -12,6 +12,10 @@
       rel="stylesheet"
       href="https://unpkg.com/tailwindcss@1.2.0/dist/tailwind.min.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins:300,400,600|Rubik&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
Note: This will be the third time we've tried to add support for fonts.

Attempt 1:
https://github.com/Recidiviz/covid19-dashboard/pull/2/files#diff-639894c3fb4e775b55e4069213b8fa69R4-R6

Attempt 2 (not merged yet):
https://github.com/Recidiviz/covid19-dashboard/pull/3/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR9-R12

I think this adding this is correct thought. What we should do:
1. Remove Attempt 1, since it's not working. We can do this at a later time.
2. Whoever merges the second of either this PR or the PR for Attempt 2 should make sure we're not adding the `<link>` element twice.

Reply:
yeah I based this on @notfelineit 's work in #3 (had to add some additional weights though) so agreed